### PR TITLE
Make simulation work with double mallet setup

### PR DESCRIPTION
**Pull request by iibrahimli**
Update the code so that simulation can be run using the double flex mallet

Currently there are a few problems:
- The second mallet (and its box) are not visible in Gazebo (though visible in RViz)
- Planning complains about the mallet finger joint violating path tolerance:
  ```
  Controller 'MalletHolder/trajectory_controller' failed with error PATH_TOLERANCE_VIOLATED: mallet_finger path error -0.763095
  [ WARN] [1696428286.555169181, 29.061000000]: Controller handle MalletHolder/trajectory_controller reports status ABORTED
  ```
These are probably related.

### CODE REVIEW

**Comment by iibrahimli** [comment]
Update: 2nd mallet now appears in the simulation, but is not controlled yet (dangles freely). The issue was lack of `transmission` blocks for the `mallet_finger` joint and `inertial` blocks for the links.

File: marimbabot_planning/src/planning.cpp [hunk 1]
```diff
@@ -33,7 +33,7 @@ Planning::Planning(const std::string planning_group) :
         }
     };
     // Create timer that checks if an action is active and moves back to the home position if not
-    auto timer = nh_.createTimer(ros::Duration(0.5), timer_callback);
+    auto timer = nh_.createTimer(ros::Duration(1000.0), timer_callback);
```
Comment (Flova) [inline marimbabot_planning/src/planning.cpp:36]: This should not be on the main, as it deactivates the homing

**Comment by Flova** [comment]
Whats the state of this? Also regarding the simulation video @iibrahimli

**Comment by Flova** [comment]
Ping @iibrahimli

**Comment by iibrahimli** [comment]
Planning to finish sim video & paper section today @Flova. Re: homing: yes my bad was testing something in sim and forgot to revert, will fix before merging this PR today

**Comment by iibrahimli** [comment]
Simulation demo video: https://cloud.mafiasi.de/s/9SiANFARrNm2F6W

**Pull request merged**